src/LiveSplit.SpeedGuidesLive/src/SGLTextEditor.cs

```diff
@@ -37,6 +38,21 @@ public SGLTextEditor()
 
             SizeChanged += SGLTextEditor_SizeChanged;
             LocationChanged += SGLTextEditor_LocationChanged;
+
+            editorTextBox.TextChanged += EditorTextBox_TextChanged;
+            webBrowser.Navigate("about:blank");
+        }
+
+        private void EditorTextBox_TextChanged(object sender, EventArgs e)
+        {
+            string html = "<html><head><style>";
+            html += "img{max-width:100%;}";
+            html += "pre{word-wrap:break-word;}";
+            html += "</style></head><body>";
+            html += Markdown.ToHtml(EditorText);
+            html += "</body></html>";
+            HtmlDocument doc = webBrowser.Document.OpenNew(true);
+            doc.Write(html);
```

Comment:
Suggestion, rather than reassigning to html constantly. In fact you don't need to store any variable.
```suggestion
private void EditorTextBox_TextChanged(object sender, EventArgs e) =>
  webBrowser.Document.OpenNew(true).Write(
    "<html><head><style>" +
    "img{max-width:100%;}" +
    "pre{word-wrap:break-word;}" +
    "</style></head><body>" +
    Markdown.ToHtml(EditorText) +
    "</body></html>";
  );
```

Reply:
I just pushed a change using C#6 style multiline format strings (which I didn't know existed tbh, and string.Format was throwing exceptions).